core/metacling/test/TClingCallFuncTests.cxx

```diff
@@ -106,6 +106,28 @@ TEST(TClingCallFunc, FunctionWrapperVoid)
    gInterpreter->ClassInfo_Delete(GlobalNamespace);
 }
 
+TEST(TClingCallFunc, FunctionWrapperVariadic)
+{
+   gInterpreter->Declare(R"cpp(
+                           void FunctionWrapperFuncVariadic(int j, ...) {}
```

Comment:
Can we actually add some work to the function body, run it and check the result? This would give us one extra degree of certainty.